### DISCUSSION TITLE
Access local resource in windows and mac desktop apps with .net MAUI

### DISCUSSION THREAD

**Discussion by satyajit-behera** [dotnet/maui#8951] [Q&A]
Is there a way I can access the local hard drive (all folders) while writing a desktop app for windows and mac using .net MAUI. Tried looking at the docs site and nothing as such available there.

This also includes calling a desktop assembly/services/resource using CLI running in windows or mac desktop system.

Thanks

**Comment by xtuzy** [comment]
maui only for ui, if you need platform function, you need google keyword+winui3, windows, macos, c#
